README.md

```diff
@@ -44,6 +44,155 @@ $ snitch c2354d53d2 -m "Finished in 23.8 seconds."
 $ snitch c2354d53d2 --message "Finished in 23.8 seconds."
 ```
 
+## API Access
+
+### Retrieving Your API Key At Command Line
+
+```ruby
+require "snitcher/api"
+
+client = Snitcher::API::Client.new(username: "jane@example.com",
+  password: "password")
+client.api_key
+```
+
+Returns a hash with an api_key key, whose value is your api_key.
+
+### Setup
+
+```ruby
+require "snitcher/api"
+
+Snitcher::API.configure do
+  api_key: "FOO"
+end
+```
+
+Alternatively you can use the client directly:
+```ruby
+require "snitcher/api"
+
+client = Snitcher::API::Client.new(api_key: "FOO")
+```
+
+### Listing Snitches
+
+```ruby
+Snitcher.snitches
+```
+
+Returns an array of hashes. Each hash represents a snitch.
+
+### Retrieve a Single Snitch
+
+```ruby
+token = c2354d53d2
+Snitcher.snitch(token)
+```
+
+Returns a hash that represents a snitch.
+
+### Retrieve Snitches That Match a Set of Tags
+
+```ruby
+tags = ["critical", "sales"]
+Snitcher.tagged_snitches(tags)
+```
+
+Returns an array of hashes. Each hash represents a snitch.
+
+### Create a Snitch
+
+Required attributes are name and interval. Optional attributes are notes and tags.
+
+```ruby
+attributes = { "name": "Nightly User Data Backups",
+  "interval": "daily",
+  "notes": "User login and usage data",
+  "tags": ["users", "critical"]
+}
+Snitcher.create_snitch(attributes)
+```
+
+Returns a hash representing the newly-created snitch.
```

Comment:
For example, the user can call methods which will error if they typo them rather than returning nil (e.g. `snitch.tkoen` vs `snitch["token"]`). They also don't need to care about string keys vs symbol keys.